lib/index.js

```diff
@@ -45,13 +45,16 @@ function generate_google_oauth2_url () {
 
 exports.register = function googleauth (server, options, next) {
   OPTIONS = options;
+  OPTIONS.config = OPTIONS.config || {};
+  OPTIONS.config.auth = false;
```

Comment:
@Y-LyN-10 could we make this 
```
OPTIONS.config.auth = OPTIONS.config.auth || false;
```
to make this _configurable_?
Not _necessary_ for `this` PR, just a thought. 🤔